### PR TITLE
Keeping our actions up to date with Dependabot

### PR DESCRIPTION
**Pull request by tkoyama010**
### Overview

[Keeping our actions up to date with Dependabot](https://docs.github.com/en/code-security/dependabot/working-with-dependabot/keeping-your-actions-up-to-date-with-dependabot)

 
relate to https://github.com/pyvista/pyvista/pull/2800

### Details

- None

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -8,3 +8,7 @@ updates:
     labels:
       - "maintenance"
       - "dependencies"
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
```
Comment (tkoyama010) [inline .github/dependabot.yml:14]:
```suggestion
      interval: "daily"
    labels:
      - "maintenance"
      - "dependencies"
```

**Review by akaszynski — APPROVED** [review]
Great idea! LGTM.

**Comment by akaszynski** [comment]
Sorry all for the compulsive merge! Can revert if needed...

**Comment by tkoyama010** [comment]
No problem.

**Pull request merged**